src/components/views/dialogs/spotlight/usePublicRoomResults.ts

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Same here.
Also if the original hooks I wrote were not fit for purpose, feel free to edit them directly. I'm not quite sure I understand the difference between the two at first glance so it might be best to consolidate over one.

What do you think?

Reply:
I’m still thinking about the best way to have the debounce and refresh logic separate from the search hooks itself, but also separate from the actual dialog itself. I’ve thought about it over the weekend and I’ll improve that by tomorrow :)